unity-client/Assets/Scripts/MainScripts/DCL/Components/UI/UIShape.cs

```diff
@@ -42,6 +42,10 @@ public float GetScaledValue(float parentSize)
         {
             if (type == Unit.PIXELS) return value;
 
+            // Prevent division by zero
+            if (parentSize <= Mathf.Epsilon)
+                parentSize = 1;
```

Comment:
This is an edge case, but wouldnt it be better to set `parentSize` to `Mathf.Epsilon`?